docs/source/platform/schema-validation.md

```diff
@@ -64,8 +63,8 @@ const getUserResolver = (root, args, context) => {
 const resolvers = {
   Query: {
     getUser: getUserResolver,
-    user: getUserResolver,
-  },
+    user: getUserResolver
```

Comment:
why are we removing the trailing commas?

Reply:
Prettier!